shoes-core/spec/shoes/star_spec.rb

```diff
@@ -202,4 +202,16 @@
       expect(subject.redraw_height).to eq(108)
     end
   end
+
+  describe "center_point" do
+    subject { Shoes::Star.new(app, parent, 100, 100, 5, 50, 30, center: false, strokewidth: 0) }
+
+    it "should return the center point of the star" do
+      expect(subject.center_point.to_a).to eq([150, 150])
+    end
+
+    it "should return the coordinates as a Point" do
+      expect(subject.center_point.class).to eq(Shoes::Point)
```

Comment:
If we do the earlier comparison via `Shoes::Point`, then this check might become unnecessary.

Reply:
Agree